docs/guide/en/form-model.md

```diff
@@ -0,0 +1,351 @@
+# Form model
+
+Form model is an abstraction over HTML forms. While you can use [forms](https://github.com/yiisoft/form) directly, 
+it can be more convenient to define form in object-oriented style. With this approach, form fields are defined as class
+properties. Besides describing form data, form model also handles presentation and validation aspects.
+
+To define a form model, create a class extending from `Yiisoft\FormModel\FormModel`.
+
+## Properties
+
+Form model properties are defined as class properties. You can also add getters and setters for each of them. 
```

Comment:
Why need getters/setters? We can just make properties public.

Reply:
`can` already means that it's not mandatory.